client/src/pages/home.jsx

```diff
@@ -22,6 +23,23 @@ const Home = () => {
         }}
         color="orange"
       />
+      <Flex mt="5vh" mb="1.8vh" px="2vw" width="full">
+        <Heading width="full" size="lg">
+          Clusters
+        </Heading>
+      </Flex>
+      <Cluster
```

Comment:
Should we put these in a loop?
Something like:
`[user.roles + user.follows].forEach(c => <Cluster ...c.data />)`
Or maybe we can do that once the api is ready

Reply:
Yep, let's do it when we have the API